docs/testplan/Smartswitch-test-plan.md

```diff
@@ -0,0 +1,876 @@
+# Test plan for Smartswitch
+
+- [Introduction](#introduction)
+- [Scope](#scope)
+- [Definitions and Abbreviations](#definitions-and-abbreviations)
+- [Test Cases](#test-cases)
+    - [1.1 Check platform inventory](#11-check-platform-inventory)
+    - [1.2 Check platform voltage](#12-check-platform-voltage)
+    - [1.3 Check platform temperature](#13-check-platform-temperature)
+    - [1.4 Check dpu console](#14-check-dpu-console)
+    - [1.5 Check midplane ip address between NPU and DPU](#15-check-midplane-ip-address-between-NPU-and-DPU)
+    - [1.6 Check DPU shutdown and power up individually](#16-check-DPU-shutdown-and-power-up-individually)
+    - [1.7 Check removal of pcie link between npu and dpu](#17-check-removal-of-pcie-link-between-npu-and-dpu)
+    - [1.8 Check the NTP date and timezone between DPU and NPU](#18-check-the-ntp-date-and-timezone-between-dpu-and-npu)
+    - [1.9 Check the State of DPUs](#19-check-the-state-of-dpus)
+    - [1.10 Check the Health of DPUs](#110-check-the-health-of-dpus)
+    - [1.11 Check reboot cause history](#111-check-reboot-cause-history)
+    - [1.12 Check the DPU state after OS reboot](#112-check-the-dpu-state-after-os-reboot)
+
```

Comment:
Hi @nissampa,
Do you want to mention the fwutil test alignment in the test plan? We have the section in HLD: https://github.com/sonic-net/SONiC/blob/4e9a891aefc880379ade617770d75fb53a07f757/doc/smart-switch/pmon/smartswitch-pmon.md?plain=1#L981-L1022

Reply:
This is just first phase of test plan. Fwutil will be addressed in the next phase.

Reply:
Got it, thanks.

Reply:
@nissampa can you add a section in HLD and put mention the same?

Reply:
Sorry, I didnt get it. What needs to be added in HLD ?